test/presenters/take_part_presenter_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class TakePartPresenterTest < ActiveSupport::TestCase
+  include ActionView::Helpers::UrlHelper
+
+  test 'presents the basic details of a content item' do
+    assert_equal take_part['description'], presented_take_part.description
+    assert_equal take_part['format'], presented_take_part.format
+    assert_equal take_part['locale'], presented_take_part.locale
+    assert_equal take_part['title'], presented_take_part.title
+    assert_equal take_part['details']['body'], presented_take_part.body
+  end
+
+private
+
+  def presented_take_part(overrides={})
```

Comment:
Can you add spaces around the `=`, per the style guide?

I am quickly becoming the human embodiment of our style guide. :blush: